yearn/v2/strategies.py

```diff
@@ -96,8 +100,96 @@ def load_harvests(self):
 
     @property
     def harvests(self) -> List[int]:
-        self.load_harvests()
+        if len(self._harvests) == 0:
+            self.load_harvests()
         return self._harvests
+        
+    @property
+    def harvests_data(self):
+        if len(self._harvests) == 0:
+            self.load_harvests()
+        sorted_harvests = sorted(self._harvests_data, key=lambda k: k['block_number'], reverse=True)
+        return sorted_harvests
+
+    @property
+    def debt_ratio(self):
+        return self.vault.vault.strategies(self.strategy.address).dict()['debtRatio'] / 1e4
```

Comment:
you can access struct items without converting to dict, contracts also work where address is expected
```suggestion
        return self.vault.vault.strategies(self.strategy)['debtRatio'] / 1e4
```